GitUp/Application/AppDelegate.m

```diff
@@ -550,17 +553,19 @@ - (IBAction)viewIssues:(id)sender {
 }
 
 - (IBAction)showAboutPanel:(id)sender {
+  NSString *version = nil;
 #if DEBUG
-  _versionTextField.stringValue = @"DEBUG";
+  version = @"DEBUG";
 #else
   if (_updatePending) {
-    _versionTextField.stringValue = NSLocalizedString(@"Update Pending", nil);
+    version = NSLocalizedString(@"Update Pending", nil);
   } else {
-    _versionTextField.stringValue = [NSString stringWithFormat:NSLocalizedString(@"Version %@ (%@)", nil), [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleShortVersionString"], [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleVersion"]];
+    version = [NSString stringWithFormat:NSLocalizedString(@"Version %@ (%@)", nil), [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleShortVersionString"], [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleVersion"]];
   }
 #endif
-  _copyrightTextField.stringValue = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"NSHumanReadableCopyright"];
-  [_aboutPanel makeKeyAndOrderFront:nil];
+  self.aboutPanel.versionString = version;
+  self.aboutPanel.copyrightString = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"NSHumanReadableCopyright"];
+  [self.aboutPanel makeKeyAndOrderFront:nil];
```

Comment:
Let's try and move some of these smarts into the new `AboutPanel` class rather than have the AppDelegate controlling this logic.

Reply:
Do you want to move all these smarts into About window?
Yes, it is possible since this logic is all around `mainBundle`.

Reply:
Preferrably. I like the AppDelegate to mainly be responsible for App level issues. So what can be moved to AboutPanel seems preferable imo.

Reply:
@lucasderraugh fixed, I guess.